pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -146,6 +146,12 @@ class BatteryModelOptions(pybamm.FuzzyDict):
                 Sets the model to include a single active particle size or a
                 distribution of sizes at any macroscale location. Can be "single"
                 (default) or "distribution". Option applies to both electrodes.
+            * "PE degradation": str
+                Set the PE degradation submodel (core-shell) to be used. Options are:
```

Comment:
can you write "Positive Electrode" in these docs so people know what "PE" means

Reply:
We need to agree on a name for this option, for compatibility for a planned acid dissolution model